cmake/Modules/FindNCCL.cmake

```diff
@@ -1,28 +1,20 @@
-# Find the nccl libraries
+# - Try to find NCCL
```

Comment:
And the two FindNCCL.cmake histories are

Caffe2 - multiple commits Aug 2016 - April 2018
https://github.com/pytorch/pytorch/commits/108f5c197fa71883948ea95ec0d5a609e737c066/cmake/Modules/FindNCCL.cmake

PyTorch - two commits Nov 2017 - Apr 2018
https://github.com/pytorch/pytorch/commits/108f5c197fa71883948ea95ec0d5a609e737c066/torch/lib/THD/cmake/FindNCCL.cmake

Looks like @pietern , and @soumith touched these. Also any thoughts on which should be used going forward?